index.html

```diff
@@ -874,7 +869,33 @@
             <code id="idl-def-PushPermissionStatus.default">default</code>
           </td>
           <td>
-            The <a>webapp</a> needs to ask for permission in order to use Push API.
+            The <a>webapp</a> needs to ask for permission in order to use the Push API.
+          </td>
+        </tr>
+      </table>
+    </section>
+    <section>
+      <h2>
+        Exceptions
+      </h2>
+      <p>
+        The Push API uses the following new <a>DOMException</a> names.
```

Comment:
Afaict it would not be a major problem to have a duplicate definition for a short time, see Anne's comment [1]. But of course we should clean it up once that situation arises. I filed an issue on the Push API spec for this: https://github.com/w3c/push-api/issues/107

[1] https://www.w3.org/Bugs/Public/show_bug.cgi?id=23033#c12